### PR TITLE
Exchange: April released features and bug fixes

### PR DESCRIPTION
**Pull request by wwarner-salesforce**

### CODE REVIEW

File: modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc [hunk 1]
```diff
@@ -7,16 +7,22 @@ endif::[]
...
 * Exchange events, such as the most recent asset details page visited in Exchange, are now shown in the Recent Items list in the common Anypoint Platform navigation bar. (EXC-3264)
 * Users can now delete a draft of portal documentation. Previously users could only delete by editing the draft, removing all new edits, and publishing. When there were multiple changes in the draft, it could be difficult to tell which edits belonged to the draft and which to the old version. With the new option to delete a draft, users can have confidence that the right changes will be discarded automatically, preserving the original documentation content. (EXC-3116)
+* Add information messages when the page name validation highlight is displayed to explain why the validation failed and how the user can make the page name valid. (EXC-3041)
+* API Manager can now access and apply custom API policies even if Exchange is down. (EXC-2680)
+* All policy URLs now have MuleSoft domains, so whitelists for API policies now require only MuleSoft URLs and not Amazon S3 URLs. (EXC-2680)
```
Comment (kuntald) [inline modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]:
```suggestion
* API Manager can now access and apply custom API policies even if Exchange is down. (EXC-2680)
>> Remove this line. This is expected behavior. No need to point it out.
```
* All policy URLs now have MuleSoft domains, so whitelists for API policies now require only MuleSoft URLs. No need to also add Amazon S3 URLs. (EXC-2680)

Comment (wwarner-salesforce) [reply modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: Changed. Thank you!

File: modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc [hunk 2]
```diff
@@ -7,16 +7,21 @@ endif::[]
 
 === New Features
 
+* Adding multiple tags to an asset is now easier. After a user types the tag and presses enter, a new tag is automatically started. (EXC-3431)
 * Exchange events, such as the most recent asset details page visited in Exchange, are now shown in the Recent Items list in the common Anypoint Platform navigation bar. (EXC-3264)
 * Users can now delete a draft of portal documentation. Previously users could only delete by editing the draft, removing all new edits, and publishing. When there were multiple changes in the draft, it could be difficult to tell which edits belonged to the draft and which to the old version. With the new option to delete a draft, users can have confidence that the right changes will be discarded automatically, preserving the original documentation content. (EXC-3116)
+* Add information messages when the page name validation highlight is displayed to explain why the validation failed and how the user can make the page name valid. (EXC-3041)
```
Comment (mcampo) [inline modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]:
```suggestion
* Add information messages when trying to create a page with an invalid name in the Asset Detail Page to explain why the validation failed and how the user can make the page name valid. (EXC-3041)
```

Comment (wwarner-salesforce) [reply modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: Looks good. Thank you!

File: modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc [hunk 3]
```diff
@@ -7,16 +7,21 @@ endif::[]
...
 
 === Fixed in this Release
 
+* Exchange now generates the correct representation of mocking service URLs for OAS specifications. (EXC-3478)
 * To prevent hackers from accessing the authorization token from the request header, the authorization token is no longer included in the request header in each request made with API Console. (EXC-3416)
-* APIs in the public portal can now be filtered by asset types. Previously all assets were shown, disregarding type filter selection. (EXC-3400)
+* Enabled filtering by asset type in the public portal. (EXC-3400, EXC-3390)
 * If the user creates a Terms and Conditions page to supplement their asset documentation page, the Terms and Conditions page is now visible at all times. (EXC-3383)
 * Add `jpeg` to the list of supported extensions for portal resources (`bmp`, `gif`, `ico`, `jpe`, `jpeg`, `jpg`, `png`, `svg`, `tif`, `tiff`, `webp`). (EXC-3351)
+* If the user selects the "API is behind firewall" option, navigates away, and returns, the selection is now remembered. (EXC-3335)
```
Comment (valkyrie69) [inline modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: This is UI so should be *API is behind firewall* option...

File: modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc [hunk 4]
```diff
@@ -7,16 +7,21 @@ endif::[]
 
 === New Features
 
+* Adding multiple tags to an asset is now easier. After a user types the tag and presses enter, a new tag is automatically started. (EXC-3431)
 * Exchange events, such as the most recent asset details page visited in Exchange, are now shown in the Recent Items list in the common Anypoint Platform navigation bar. (EXC-3264)
 * Users can now delete a draft of portal documentation. Previously users could only delete by editing the draft, removing all new edits, and publishing. When there were multiple changes in the draft, it could be difficult to tell which edits belonged to the draft and which to the old version. With the new option to delete a draft, users can have confidence that the right changes will be discarded automatically, preserving the original documentation content. (EXC-3116)
+* Add information messages when trying to create a page with an invalid name in the Asset Detail Page to explain why the validation failed and how the user can make the page name valid. (EXC-3041)
```
Comment (valkyrie69) [inline modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: Added information messages...

File: modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: Thank you! Fixed.

File: modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc (hunk 4 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: Thank you! Fixed.

**Pull request merged**